data/json/items/melee/bludgeons.json

```diff
@@ -1764,5 +1764,62 @@
     "price": 16000,
     "qualities": [ [ "HAMMER", 1 ] ],
     "melee_damage": { "bash": 22, "stab": 23 }
+  },
+  {
+    "type": "GENERIC",
+    "id": "exodii_club",
+    "name": "crusher",
+    "description": "Resembling a tetsubo or baseball bat studded with metal spikes, this is a simple and brutal weapon used by Exodii when the ammo runs low. The massive weight makes this a daunting weapon to wield for those without artificial muscles.",
+    "weight": "10 kg",
+    "min_strength": 12,
+    "price_postapoc": 400,
+    "color": "grey",
+    "symbol": "|",
+    "material": "qt_steel",
+    "techniques": [ "WBLOCK_1", "SPIN", "WIDE", "BRUTAL", "SWEEP" ],
+    "flags": [ "DURABLE_MELEE", "SHEATH_SPEAR", "ALWAYS_TWOHAND" ],
+    "weapon_category": [ "MACES", "GREAT_HAMMERS" ],
+    "volume": "2250 ml",
+    "longest_side": "120 cm",
+    "to_hit": { "grip": "weapon", "length": "long", "surface": "any", "balance": "uneven" },
+    "qualities": { "HAMMER": 1 },
+    "melee_damage": { "bash": 40, "pierce": 20 }
+  },
+  {
+    "id": "exodii_powerfist",
+    "type": "ARMOR",
+    "name": { "str_sp": "hammerhand" },
+    "description": "A weapon used by bipedal Exodii when the ammo runs low. Delivering a punch with this glove triggers a pneumatic hammer that slams into the target. A built-in compressor requires a few seconds between strikes.",
```

Comment:
Rather than a compressor I suggest a winch and spring mechanism, I think it's got fewer moving parts and I love the idea of the exodii using weaponized springs a lot.

Reply:
Yeah that's a better feel, I'll change it up.